<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="left" native="true">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="titleMenuList" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_1">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="labelTitle">
             <property name="text">
              <string>Assets</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustIgnored</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>244</width>
               <height>327</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>650</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item alignment="Qt::AlignTop">
                 <widget class="QGroupBox" name="groupBox">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>450</height>
                   </size>
                  </property>
                  <property name="title">
                   <string/>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <spacer name="verticalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeType">
                      <enum>QSizePolicy::Fixed</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QWidget" name="menuFileContainer" native="true">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>160</height>
                      </size>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_2">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>20</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>20</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item alignment="Qt::AlignTop">
                       <widget class="QPushButton" name="pushButtonTokenSend">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>50</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>16777215</width>
                          <height>50</height>
                         </size>
                        </property>
                        <property name="focusPolicy">
                         <enum>Qt::NoFocus</enum>
                        </property>
                        <property name="text">
                         <string>Send Token</string>
                        </property>
                        <property name="checkable">
                         <bool>true</bool>
                        </property>
                        <property name="autoExclusive">
                         <bool>false</bool>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QWidget" name="fileButtonsWidget" native="true">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>165</height>
                      </size>
                     </property>
                     <layout class="QVBoxLayout" name="verticalLayout_6">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QWidget" name="menuToolsContainer" native="true">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>160</height>
                      </size>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_3">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>20</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>20</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item alignment="Qt::AlignTop">
                       <widget class="QPushButton" name="pushButtonTokens">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>50</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>16777215</width>
                          <height>50</height>
                         </size>
                        </property>
                        <property name="focusPolicy">
                         <enum>Qt::NoFocus</enum>
                        </property>
                        <property name="text">
                         <string>View Tokens</string>
                        </property>
                        <property name="checkable">
                         <bool>true</bool>
                        </property>
                        <property name="autoExclusive">
                         <bool>false</bool>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QWidget" name="configurationButtonsWidget" native="true">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>105</height>
                      </size>
                     </property>
                     <layout class="QVBoxLayout" name="verticalLayout_7">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="leftMargin">
                      <number>20</number>
                     </property>
                     <property name="rightMargin">
                      <number>20</number>
                     </property>
                     <item alignment="Qt::AlignTop">
                      <widget class="QPushButton" name="pushButtonTokenUsernames">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>50</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>50</height>
                        </size>
                       </property>
                       <property name="focusPolicy">
                        <enum>Qt::NoFocus</enum>
                       </property>
                       <property name="text">
                        <string>View Usernames</string>
                       </property>
                       <property name="checkable">
                        <bool>true</bool>
                       </property>
                       <property name="autoExclusive">
                        <bool>false</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QWidget" name="optionsButtonsWidget" native="true">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>160</height>
                      </size>
                     </property>
                     <layout class="QVBoxLayout" name="verticalLayout_8">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalTools">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="leftMargin">
                      <number>20</number>
                     </property>
                     <property name="rightMargin">
                      <number>20</number>
                     </property>
                     <item alignment="Qt::AlignTop">
                      <widget class="QPushButton" name="pushButtonTokenHistory">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>50</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>50</height>
                        </size>
                       </property>
                       <property name="focusPolicy">
                        <enum>Qt::NoFocus</enum>
                       </property>
                       <property name="text">
                        <string>View Token History</string>
                       </property>
                       <property name="checkable">
                        <bool>true</bool>
                       </property>
                       <property name="autoExclusive">
                        <bool>false</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QWidget" name="toolsButtonsWidget" native="true">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>160</height>
                      </size>
                     </property>
                     <layout class="QVBoxLayout" name="verticalTools_11">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalHelp">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="leftMargin">
                      <number>20</number>
                     </property>
                     <property name="rightMargin">
                      <number>20</number>
                     </property>
                     <item alignment="Qt::AlignTop">
                      <widget class="QPushButton" name="pushButtonTokenCreate">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>50</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>50</height>
                        </size>
                       </property>
                       <property name="focusPolicy">
                        <enum>Qt::NoFocus</enum>
                       </property>
                       <property name="text">
                        <string>Create Token</string>
                       </property>
                       <property name="checkable">
                        <bool>true</bool>
                       </property>
                       <property name="autoExclusive">
                        <bool>false</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QWidget" name="helpButtonsWidget" native="true">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>100</height>
                      </size>
                     </property>
                     <layout class="QVBoxLayout" name="verticalHelp">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="right" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QStackedWidget" name="stackedWidgetContainer"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
